pkg/controller/attach_detach.go

```diff
@@ -496,8 +497,11 @@ func (r *ReconcileAttachDetach) Recover(ctx context.Context) error {
 	}
 	// retrigger any aborted operation from possible previous controller crash
 	recovered := &sync.Map{}
+	// Make Recover Annotation unique per Recovery, so if driver restarts more than once
+	// the AzVolumeAttachment can still get a new Recover Annotation to force register an UPDATE event for reconciliation
+	uniqueRecoverAnnotation := uuid.NewUUID().String()
```

Comment:
Consider creating the recovery uuid in `azuredisk_v2.go` and passing it in as an argument to `attachdetach.Recover`, `azvolume.Recover`, and `azdrivernode.Recover`.

Reply:
np: Consider naming the variable `recoveryId` maybe?

Reply:
Like the idea to name it `recoveryId `! Also if we create the uuid in `azuredisk_v2.go`, we can use the same uuid for 3 controllers. Nice!